Gruntfile.js

```diff
@@ -602,47 +603,54 @@ module.exports = function (grunt) {
 
     // Test settings
     karma: {
-      unit: {
+      options: {
         configFile: 'test/karma.conf.js',
-        singleRun: true,
         // default in karma.conf.js is Firefox, however, Chrome has much better
         // error messages when writing tests.  Call like this:
         // grunt test
         // grunt test --browsers=Chrome
         // grunt test --browsers=Chrome,Firefox,Safari (be sure karma-<browser_name>-launcher is installed)
         browsers: grunt.option('browsers') ?
                     grunt.option('browsers').split(',') :
-                    ['Firefox']
+                    // if running locally on mac, we can test both FF & Chrome,
+                    // in Travis, just FF
+                    // ['Nightmare'] is a good alt for a current headless
+                    // TODO: fix this, PhantomJS is deprecated
```

Comment:
Let's track as an issue instead of a TODO in the code

Reply:
ok ill update to FIXME & open an issue.

Reply:
I'd remove the comment altogether and just use the issue. I'd prefer to track things once, and TODOs tend to get forgotten